### DISCUSSION TITLE
Downloaded dictionary files appear to be corrupted?

### DISCUSSION THREAD

**Discussion by Starhowl** [birchill/10ten-ja-reader#609] [General]
I've installed the latest Developer Edition of Firefox and enabled `xpinstall.signatures.required` in `about:config` but I'm still getting told that every dictionary file appears to be corrupt.

Is there something that I'm missing here?

**Comment by birtles** [comment]
Oh, that's odd. Can you show me a screenshot of the error?

**Reply by Starhowl** [reply]
[A picture of the error message.](https://i.imgur.com/bJ5sO4x.png)

**Reply by birtles** [reply]
That seems to be related to rikaichan, not this add-on (rikaichamp).

(Also, Rikaichamp doesn't require you to download dictionary data files. It includes a basic words data file pre-packaged and then downloads the latest files for the words and other dictionaries in the background.)

**Reply by Starhowl** [reply]
Do you know which (e.g. names, Japanese-English, etc.) dictionaries Rikaichamp includes?

**Reply by birtles** [reply]
Yes, you can see them in the options page.

It includes:
* A words dictionary (JMdict/EDICT) which is Japanese-English but you can include different translations of the glosses so you have Japanese-French/English or Japanese-Russian/English etc.
* A names dictionary (JMNedict/ENAMDICT) which is strictly Japanese-English.
* A kanji dictionary (KANJIDIC) which is Japanese-English but also offers loading other translations of the glosses.

These dictionaries are automatically updated every week by downloading just the updated entries.

There is also other data mixed in like pitch accent data, kanji kentei data, conning references, revised kanji component data etc.

![image](https://user-images.githubusercontent.com/1232595/119220307-de6f8780-bb24-11eb-9258-72f6b7f8a0ab.png)

![image](https://user-images.githubusercontent.com/1232595/119220319-ecbda380-bb24-11eb-86c6-5ace2cd97ef0.png)
